src/pipeline/server.rs

```diff
@@ -186,6 +186,13 @@ where
     type Output = Result<(), Error<T, S>>;
 
     fn poll(self: Pin<&mut Self>, cx: &mut Context) -> Poll<Self::Output> {
+        #[cfg(feature = "tracing")]
+        let span = tracing::trace_span!("poll");
```

Comment:
IMO, this should probably be 
```suggestion
        let span = tracing::trace_span!("Server::poll");
```
or something. otherwise, there are a lot of polls in the world...

Reply:
It'll be annotated with the module though, no?

Reply:
It's up to the formatter whether or not targets are displayed, they can be turned on or off by the user...

Reply:
I guess the question is why shouldn't it then be `multiplex::Server::poll`? Or `tokio_tower::multiplex::Server::poll`? I'm not sure where the line goes for what should and should not be included in the span name. My immediate instinct was to make it local to the module..